main/streams/plain_wrapper.c

```diff
@@ -143,7 +143,7 @@ typedef struct {
 } php_stdio_stream_data;
 #define PHP_STDIOP_GET_FD(anfd, data)	anfd = (data)->file ? fileno((data)->file) : (data)->fd
 
-static int do_fstat(php_stdio_stream_data *d, int force)
+static zend_bool do_fstat(php_stdio_stream_data *d, int force)
```

Comment:
I am not sure, zend_bool here is right.  the underlying fstat return -1(TRUE) or error and return 0(FALSE) on success, -1, 0 is not a typical bool values...